style.css

```diff
@@ -541,3 +557,288 @@ textarea:focus {
   border-radius: 10px;
   background-color: #0d151e;
 }
+
+/* Media Query */
+
+/* Mobile Screen less than 300px */
+
+@media screen and (max-width: 300px) {
+  .see-project .p {
+    font-size: 0.8rem;
+  }
+
+  .see-resume .p {
+    font-size: 0.75rem;
+  }
+}
+
+/* Desktop view 992px */
+
+@media screen and (min-width: 992px) {
+  .Ham-menu {
+    display: none;
+  }
+
+  .menu {
+    width: 25%;
+  }
+
+  .desktop-menu {
+    display: block;
+    width: 100%;
+  }
+
+  .desktop-menu li {
+    list-style-type: none;
+  }
+
+  .framework-icon li,
+  .lang-icon li,
+  .skill-icon li {
+    padding: 10px;
+    height: 100%;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+  }
+
+  .desktop-menu ul li:hover {
+    cursor: pointer;
+  }
+
+  .desktop-menu li a {
+    text-decoration: none;
+    color: #344563;
+    font-weight: bolder;
+    font-size: 0.9rem;
+  }
+
+  .desktop-menu ul {
+    display: flex;
+    justify-content: end;
+    align-items: center;
+    gap: 32px;
+  }
+
+  .hero {
+    position: relative;
+    background-color: #fff;
+    background-image: none;
+    height: 83vh;
+    border-bottom-left-radius: 25% 100px;
+    overflow: hidden;
+  }
+
+  .desktop-bg {
+    display: block;
+    z-index: 1;
+    background-image: url(./images/DesktopheaderBg.png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    height: 524px;
+    width: 93%;
+    margin-top: 90px;
+    margin-right: 8px;
+    position: absolute;
+    overflow: hidden;
+  }
+
+  header {
+    position: fixed;
+    z-index: 100;
+    width: 100%;
+    padding: 25px 40px;
+    background-color: #fff;
+  }
+
+  .banner {
+    z-index: 2;
+    margin-top: 135px;
+    width: 70%;
+  }
+
+  .project-img img {
+    display: none;
+  }
+
+  .social-icon li img {
+    height: 50px;
+    width: 50px;
+  }
+
+  .project-img {
+    height: 100%;
+  }
+
+  .project-show-case {
+    width: 95%;
+    margin: 0 auto;
+    margin-top: 114px;
```

Comment:
let's fix this there's too much top margin from the project-show-case section please reduce it to fit the actual design.
your design
![mar](https://user-images.githubusercontent.com/53356820/144390507-f0913ce4-d02b-4d99-ab96-2448afaba0d1.png)

Reply:
I have resolved the issue. thanks for pointing it out.